Client/src/components/moviePageNav.tsx

```diff
@@ -0,0 +1,41 @@
+import {useState} from 'react'
+
+type MoviePageNavProps ={
+    active:string
+    setActive: React.Dispatch<React.SetStateAction<string>>
+}
+
+const MoviePageNav = (props:MoviePageNavProps) => {
+
+    const handleClick = (value:string) => {
+            props.setActive(prev => prev = value)
+    }
+
+
+    return(
+            <div className="movieNav">
+                    <ul>
+                        <li 
+                            style={{borderBottom: props.active === 'rec' ? ' #782CBF 4px solid' : "none"}}
```

Comment:
może to zrobić jako help function do stylowania?

Reply:
Nie za bardzo wiem jak to zrobić, w sensie jak stworzyć funkcje w scss która przyjmie parametry i wywołać ją w pliku tsx.

Reply:
`
const getBorderStyles = (active) => {
  const actives = ['rec', 'det', 'cast'];
  if (actives.includes(active)) {
    return '#782CBF 4px solid';
  }
  return 'none';
}
`

Reply:
Zrobione